.circleci/config.yml

```diff
@@ -48,6 +48,9 @@ jobs:
       - run:
           command: pip install -e .
           name: setup
+      - run:
+          name: extra
+          command: pip3 install termcolor
```

Comment:
Can we rename it to something better than "extra"? This won't be so informative. Maybe something like `install termcolor` or something?